### ISSUE TITLE
Restricted access periods for archival data

### ISSUE THREAD

**Issue by ybanens** [GriffithUniLibrary/intro-text-mining-analysis#5]
Do we need to mention Queensland Government Restricted Access Periods (RAPs) when discussing historical data with personally identifiable information in it? See https://www.forgov.qld.gov.au/information-and-communication-technology/recordkeeping-and-information-management/recordkeeping/transfer-and-manage-records-at-qsa/restrict-access-to-records-at-qsa-restricted-access-periods

**Comment by stapletonsl** [comment]
Good idea to add a sentence about it, useful for researchers working with archival data.

**Comment by ybanens** [comment]
Closed without being done. Reopening until done.
